packages/analytics-js/src/components/capabilitiesManager/CapabilitiesManager.ts

```diff
@@ -144,7 +145,12 @@ class CapabilitiesManager implements ICapabilitiesManager {
       state.capabilities.isOnline.value = true;
     });
 
-    // TODO: add debounced listener for globalThis.onResize event and update state.context.screen.value
+    globalThis.addEventListener('resize', () => {
+      // Adding a timeout as resize event can fire frequently
+      setTimeout(() => {
```

Comment:
thi will create too many timeouts as it will not debounce, but just delay. please use a debounce utility function like https://www.freecodecamp.org/news/javascript-debounce-example/

Reply:
Thanks. Addressed.